src/main/java/io/anserini/index/IndexDenseVectors.java

```diff
@@ -364,19 +372,19 @@ public Counters run() throws IOException {
   }
 
   public static void main(String[] args) throws Exception {
-    IndexVectorArgs indexCollectionArgs = new IndexVectorArgs();
+    IndexDenseVectorsArgs indexCollectionArgs = new IndexDenseVectorsArgs();
```

Comment:
a java question

I'm a bit unsure how to properly initialize the Args here if we make it subclass
@lintool 
I don't think I can do
```
IndexDenseVectors.Args args = new IndexDenseVectors.Args()
```

Reply:
we need to parse the args first and then initialize the index class as IndexDenseVectors(args)